spec/models/workflow_spec.rb

```diff
@@ -15,6 +15,8 @@
     let(:translatable) { create(:workflow_with_contents, build_extra_contents: true) }
     let(:translatable_without_content) { build(:workflow, build_contents: false) }
     let(:primary_language_factory) { :workflow }
+    let(:private_project) { create(:project, private: true) }
```

Comment:
that link is a ref's a private_model a few lines above and not the private_project here.